src/lib.rs

```diff
@@ -283,8 +283,9 @@ impl IronOxide {
         // create a tokio runtime with the default number of core threads (num of cores on a machine)
         // and an elevated number of blocking_threads as we expect heavy concurrency to be network-bound
         let runtime = tokio::runtime::Builder::new()
-            .blocking_threads(250) // most all SDK methods will block on the network
-            .keep_alive(None)
+            .threaded_scheduler()
+            .enable_all()
+            .max_threads(250) // most all SDK methods will block on the network
```

Comment:
Maybe just for me, but it might be nice to comment each of these lines to explain what they mean. Or link to docs about it? Feel free to ignore if this is something obvious.

Reply:
Commenting the lines for now seems reasonable to me. @giarc3, would you please?

Reply:
Done